pkg/ddevapp/config.go

```diff
@@ -167,13 +167,8 @@ func NewApp(appRoot string, includeOverrides bool) (*DdevApp, error) {
 
 	// Rendered yaml is not there until after ddev config or ddev start
 	if fileutil.FileExists(app.ConfigPath) && fileutil.FileExists(app.DockerComposeFullRenderedYAMLPath()) {
-		content, err := fileutil.ReadFileIntoString(app.DockerComposeFullRenderedYAMLPath())
-		if err != nil {
-			return app, err
-		}
-		err = app.UpdateComposeYaml(content)
-		if err != nil {
-			return app, err
+		if err := app.ReadDockerComposeYAML(); err != nil {
+			util.WarningOnce("Unable to read '%s' project config at %s: %v", app.Name, app.DockerComposeFullRenderedYAMLPath(), err)
```

Comment:
Show this error only once, since `ddevapp.NewApp()` will be called multiple times if there are problems with the project itself.

I didn't add caching for `ddevapp.GetActiveApp()` on errors because it led to unpredictable behavior and unexpected errors.